### ISSUE TITLE
Eliminate white stripe above footer on some pages

### ISSUE THREAD

**Issue by graphiclunarkid** [openrightsgroup/cmp-issues#47]
On pages where the background colour of the last content `div` is red there is a white stripe between this and the footer. It might also exist on other pages but it isn't visible there because the background of the last content `div` is white.

![white-band](https://cloud.githubusercontent.com/assets/433456/3354659/9dc089b4-faa6-11e3-8093-cfa904bb3fcd.png)

Currently affected pages:
- Home
- Unblock

Please can we get rid of it so that the background colour goes from red straight to the footer's black on these pages?

**Comment by webal** [comment]
I've added a couple of classes to the CSS to handle this .no-top-margin and .no-bottom-margin.

For the home page to work this needs to be applied to the bottom of the red box, but also as it is the last box on the page the top of the footer. See the updated git repo to see what I changed on index.html(https://github.com/openrightsgroup/blocked-org-uk/commit/f8ed556643c5607aad371a66b4960e484591c417) as you'll need some way of letting a page in modx tell the footer to add that class.
